src/viewPages/ListComments.jsx

```diff
@@ -11,6 +11,20 @@ export default function ListComments() {
   const { id_article } = useParams();
   const [articles, setArticles] = useState([]);
   const [selectedArticle, setSelectedArticle] = useState(null);
+  const [newComment, setNewComment] = useState("");
+
+  const handleAddComment = (event) => {
+    event.preventDefault();
+  
+    postApi(`/api/articles/${id_article}/comments`, {author: selectedArticle.author, body: newComment}) // the author will be by default the author of the article
```

Comment:
It would be nice if we just passed the `id_article` to `postApi` here, rather than all of the url. You can have the url to accept the parametric in your Api.js file 👍